src/components/HearingList.js

```diff
@@ -136,6 +136,17 @@ export class HearingListItem extends React.Component {
       en: 'Questionnaire available in English',
     };
 
+    // Preparing the dates for translation.
+    const isPast = function(time) { return time < new Date().getTime() };
+    const openTime = formatTime(hearing.open_at, {hour: '2-digit', minute: '2-digit'});
+    const openDate = formatDate(hearing.open_at, {day: '2-digit', month: '2-digit', year: 'numeric'});
+    const closeTime = formatTime(hearing.close_at, {hour: '2-digit', minute: '2-digit'});
+    const closeDate = formatDate(hearing.close_at, {day: '2-digit', month: '2-digit', year: 'numeric'});
+
+    // Translation ID's for ITIL translation values
+    let openMessageId = 'timeOpen' + (isPast(openTime) ? 'Past' : 'Future') + 'WithValues';
+    let closeMessageId = 'timeClose' + (isPast(openTime) ? 'Past' : 'Future') + 'WithValues';
```

Comment:
should be `isPast(closeTime)`?

Reply:
Apparently committed an usaved version of the file where this was fixed, should now be ok.